tekton/pipelineresources.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: tekton-operator-git # Replace with a new name e.g. tekton-operator-git-v0.14.2-1
+spec:
+  type: git
+  params:
+  - name: url
+    value: https://github.com/nikhil-thomas/operator
```

Comment:
This needs to change :stuck_out_tongue:

Reply:
i remembered to reset the revision, but i missed this one 😛

Reply:
:+1: